### PR TITLE
chore: remove hard-coded page caching

### PR DESCRIPTION
**Pull request by davidlougheed**

### CODE REVIEW

File: chord_metadata_service/phenopackets/api_views.py [hunk 1]
```diff
@@ -97,9 +89,6 @@ class BiosampleViewSet(ExtendedPhenopacketsModelViewSet):
     post:
     Create a new biosample
     """
-    queryset = m.Biosample.objects.all() \
-        .prefetch_related(*BIOSAMPLE_PREFETCH) \
-        .order_by("id")
```
Comment (v-rocheleau) [inline chord_metadata_service/phenopackets/api_views.py:102]: Why was this removed?

**Review by v-rocheleau — COMMENTED** [review]
Looks good, one question

File: chord_metadata_service/phenopackets/api_views.py (hunk 1 above)
Comment (davidlougheed) [reply chord_metadata_service/phenopackets/api_views.py:102]: not visible in the diff, but it's a duplicate - if you expand below you can see its there a second time

**Review by v-rocheleau — APPROVED** [review]
lgtm!
